docker-compose.yml

```diff
@@ -0,0 +1,31 @@
+# Configuration for the services that are required to run an analysis of the
+# data from riff edu mattermost.
+# see https://www.metabase.com/docs/latest/operations-guide/running-metabase-on-docker.html
+version: '3.6'
+services:
+
+  target-db:
+    image: mysql:5.7
+    container_name: mysql-business-analytics
+    environment:
+      MYSQL_ROOT_PASSWORD: root
+    ports:
+      - '3306:3306'
+    volumes:
+      - target-db-data:/var/lib/mysql
+
+  metabase:
+    image: metabase/metabase:latest
+    container_name: metabase-business-analytics
+    environment:
+      MB_DB_FILE: ./metabase-db/metabase.db
+    ports:
+      - '3000:3000'
+    volumes:
+      - ./metabase-db:/metabase-db
+    depends_on:
+      - target-db
+
+volumes:
+  target-db-data:
```

Comment:
A similar comment about volume names as I made above about service names, so keep the prefix you used for the service names, and relate the volume to the service it will be bound to, e.g. `ba-mmdb-data`

Reply:
Also makes sense, added consistent names to the volumes.